sdkproject/Assets/Mapbox/Unity/Map/AbstractMap.cs

```diff
@@ -126,7 +131,18 @@ protected virtual void Start()
 
 		void TileProvider_OnTileAdded(UnwrappedTileId tileId)
 		{
-			_mapVisualizer.LoadTile(tileId);
+			var utile = _mapVisualizer.LoadTile(tileId);
+			if (!_centerTileSet)
+			{
+				_centerTileSet = true;
+
+				// FIXME: this is never unregistered!
```

Comment:
This is about that "central tile detection" thing again. My main problem with this was, I wanted it to support central tile changes so you can shift the world in any direction. And if I remember correctly, I was thinking about creating a function to shift the world and move this to that function in the future. Then it won't be that utile object, but it'll be the one passed to that function. It might be better trying to implement that right away, even if we don't do the shifting thing though.